src/ec/suite_b/ops/ops.rs

```diff
@@ -252,13 +252,12 @@ impl CommonOps {
     // Gueron for the details. This is the case for both the field order and
     // group order for both P-256 and P-384, but it is not the case for all
     // curves. For example, it is not true for P-521.
+    //
+    // TODO(DJ): This interface is misleading: `a` is actually mutated.
```

Comment:
You're right. I was confused and thinking of `*a` as if it was taking a raw pointer, but of course it's not.